src/plugins/expressions/common/execution/execution.ts

```diff
@@ -280,13 +280,18 @@ export class Execution<
    * because in legacy interpreter it was set to `null` by default.
    */
   public start(
-    input: Input = null as any
+    input: Input = null as any,
+    isSubExpression?: boolean
```

Comment:
i see this affects our public API ...
can you move this to a private function and expose public function without isSubExpression (that just calls the private function)

Reply:
Unfortunately this cannot be easily done without changing one public function at least (either creation or execution one) as subExpressions are managed via public `execution` API